### PR TITLE
Rubocop event driven accouting feature

### PR DESCRIPTION
**Pull request by KGibb8**

### CODE REVIEW

File: app/controllers/members/two_factor_controller.rb [hunk 1]
```diff
@@ -48,7 +48,7 @@ def resend_code
         if session[:resend_two_factor_code] && validate_nonce(session[:resend_two_factor_code], 320.seconds)
           format.json { render json: { success: false, message: "Wait 5 minutes before requesting another code" } }
         else
-          session[:resend_two_factor_code] = nonce(Time.now)
+          session[:resend_two_factor_code] = nonce(Time.zone.now)
```
Comment (sadeden) [inline app/controllers/members/two_factor_controller.rb]: I think `Time.current` is doing the same. It's one method call less :)

File: app/controllers/reauthentication_controller.rb [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
   def new
-    render 'devise/sessions/reauthenticate'
+    render "devise/sessions/reauthenticate"
   end
 
   def create
     result = AuthenticatePassword.call(member: current_member, password: permitted_params[:password])
     if result.success?
-      session[:reauthenticated_at] = Time.now
+      session[:reauthenticated_at] = Time.zone.now
```
Comment (sadeden) [inline app/controllers/reauthentication_controller.rb]: Idem, `Time.current`

Comment (KGibb8) [reply app/controllers/reauthentication_controller.rb]: :+1:

File: app/controllers/members/two_factor_controller.rb (hunk 1 above)
Comment (KGibb8) [reply app/controllers/members/two_factor_controller.rb]: :+1:

**Pull request merged**